.github/CODEOWNERS

```diff
@@ -9,8 +9,15 @@
 # Resources
 #  - [CODEOWNERS] https://help.github.com/articles/about-codeowners
 
-*       @okta/eng-uicore
+*       @rajaguruduraisamy-okta @ganeshsomasundaram-okta @jmelberg-okta
```

Comment:
This change is already in master. Please rebase this branch to ensure we're only pulling in necessary commits.

Reply:
@jmelberg-okta should be fixed now! we had some unexpected history/conflict issues.